### PR TITLE
[DOC] Visualize documentation update

### LINKED ISSUE #6986: Documentation Update: list of widgets
Here's a list of outdated documentation files to keep track when updating. List based only on the images. Cannot think how to easily inspect text-based changes. Some have only minor changes. Many have only Report (or similar) moved to status bar, they aren't listed here. That said, it might make sense to update them at some point, just to have a full update every ... 10 years? :)

# Data
- [ ] File #6966
- [ ] Datasets #6967
- [ ] Data Table #6968
- [ ] SQL Table
- [ ] Paint Data #6977
- [ ] Data Info #6978
- [ ] Edit Domain
- [ ] Color (Save, Load, Reset buttons added) #6981 

# Transform
(resolved in https://github.com/biolab/orange3-doc-visual-programming/pull/7)
- [x] Data Sampler
- [x] Select Columns
- [x] Select Rows (minimal)
- [x] Transpose
- [x] Merge Data (minimal)
- [x] Select By Data Index
- [x] Aggregate Columns
- [x] Group by (some aggregations added and order changed)
- [x] Apply Domain
- [x] Impute
- [x] Continuize (wording different)
- [x] Randomize (minor)
- [x] Purge Domain (minor)
- [x] Formula (no change but VERY poor quality image)
- [x] Create Class
- [x] Python Script

# Visualize
(resolved in https://github.com/biolab/orange3-doc-visual-programming/pull/2)
- [x] Tree Viewer
- [x] Distribution
- [x] Scatter Plot
- [x] Line Plot (very minor)
- [x] Sieve Diagram (very minor)
- [x] Mosaic Display
- [x] FreeViz
- [x] RadViz
- [x] Heat Map (low-high to range)
- [x] Pythagorean Tree (very minor)
- [x] Nomogram
- [x] Scoring Sheet Viewer (should be redone after #6924 to have the same results each time)

# Model
- [x] Logistic Regression
- [x] AdaBoost
- [x] Neural Network
- [x] Stochastic Gradient Descent
- [x] Save Model

# Evaluate
- [x] Test and Score
- [x] Predictions
- [x] Confusion Matrix
- [ ] Performance Curve (minor)
- [x] Parameter Fitter (missing Example)

# Unsupervised
- [x] Distance File
- [x] Correlations
- [x] Distance Map (low-high to range)
- [x] Hierarchical Clustering
- [x] Manifold Learning (minor)
- [x] PCA (minor)
- [x] Neighbors (minor)
- [x] Distances
- [x] MDS
- [x] Save Distance Matrix

### PR DESCRIPTION
**Pull request by ajdapretnar**
##### Issue

Partially resolves #6986.

##### Description of changes
- Tree Viewer

##### Includes
- [ ] Code changes
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: doc/visual-programming/source/widgets/visualize/images/TreeViewer-regression.png
Comment (markotoplak) [inline doc/visual-programming/source/widgets/visualize/images/TreeViewer-regression.png:1]: I tried making this an indexed image in Gimp and it was 128KB (the original was 370).

File: doc/visual-programming/source/widgets/visualize/images/TreeViewer-stamped.png
Comment (markotoplak) [inline doc/visual-programming/source/widgets/visualize/images/TreeViewer-stamped.png:1]: Stamps are too small.

We should also be careful in setting visualizations not to waste space. Here, the window could be decreased in width a bit so that the widget becomes less wide. Also, the height could be possibly made smaller.

Comment (markotoplak) [reply doc/visual-programming/source/widgets/visualize/images/TreeViewer-stamped.png:1]: I checked: stamps always used to be so small... But now we have the opportunity to improve.

Comment (markotoplak) [reply doc/visual-programming/source/widgets/visualize/images/TreeViewer-stamped.png:1]: @janezd and me decided that we have to fix the stamper and that I should do it.

File: doc/visual-programming/source/widgets/visualize/images/TreeViewer-regression.png
Comment (janezd) [reply doc/visual-programming/source/widgets/visualize/images/TreeViewer-regression.png:1]: On macOS I think I used to do it with ImageMagick. It includes a command-line utility `convert`.

We were just discussing it with @markotoplak and @ajdapretnar. Perhaps we should abandon indexing altogether because it often visibly degrades the colors. This will increase the size of image files, but if we move all documentation to web (as discussed in another issue), we don't mind.

**Comment by markotoplak** [comment]
Please open a corresponding PR on the submodule.
